packages/cozy-procedures/src/components/EndModal.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withRouter } from 'react-router'
+import { withClient } from 'cozy-client'
+import {
+  Modal,
+  ModalContent,
+  ModalFooter,
+  Title,
+  translate,
+  Button,
+  ButtonLink
+} from 'cozy-ui/transpiled/react/'
+import AppLinker, {
+  generateUniversalLink
+} from 'cozy-ui/transpiled/react/AppLinker'
+import IllustrationOK from '../assets/ok.svg'
+import IllustrationError from '../assets/error.svg'
```

Comment:
Come on, is it working? What I've done when I tested it? :o

Reply:
Does this mean that cozy-procedures rely on a particular loader ? If so I am not sure that it should, since it is a library. A babel plugin could be used to inline the SVGs at transpile time. https://github.com/airbnb/babel-plugin-inline-react-svg

Reply:
Yes, i'm assuming it relies on an SVG loader. I don't know, it seems fair to be abble to import an SVG or a JSON file and that a loader would be present, but I'll look into the babel inliner plugin.

Reply:
>  I don't know, it seems fair to be abble to import an SVG or a JSON file and that a loader would be present,

For example, I am not sure importing the cozy-procedures lib in codesandbox would work in codesandbox.

Researching just a bit what create-react-app used led me to https://github.com/facebook/create-react-app/issues/5293#issuecomment-427085505, it seems create-react-app uses import names to automatically switch behavior, seems very interesting.